README.md

```diff
@@ -121,11 +121,7 @@ MySQL password should be the same for initialSetupTests as openmrs password
     ```
 You don’t need to set up an OpenMRS instance since we use a [cloud instance](https://openmrs-spa.org/openmrs/spa) for the test backend.
 
-## Running tests
-Navigate into bdd tests module
```

Comment:
instead of having too many low tickets why not have it once fixed in one single pull request.  Were you not supposed to fix it in the later ticket ?

Reply:
> Were you not supposed to fix it in the later ticket ?

I just realized it late 

and we need to have the module documentation updated for easy directive to the contributors.